docs/guides/1226-coding-style.md

```diff
@@ -381,3 +381,13 @@ client: _
 // ok if it improves understanding
 client: platform: _
 ```
+
+## Format all the CUE code
+
+It's important to lint all the CUE code so it stays consistent across all packages.
+
+Thanksfully, CUE makes it very easy. [First, make sure you have the CUE binary installed on your machine](https://cuelang.org/docs/install/). Then, simply run the following command from the package directory:
+
+```shell
+cue fmt -s
+```
```

Comment:
They can also run `make cuefmt` from the root of dagger repository, should we documente that one?
They can also run `dagger do lint cue` to only find error.

Reply:
* `dagger do lint cue` 👍
* `make cuefmt` 👎 we should remove all dependencies on our Makefile in preparation for removing it altogether

Reply:
agreed with @shykes for pushing dagger when it's about code on the dagger repo. The reason why I suggest `cue fmt -s` here is to make it work outside of our repo, I assume you may want to write new packages on any repo, so I'd say our internal tooling is out of scope here.

Reply:
Yes I agree `cue fmt -s` 👍